datasets/ttc4900/ttc4900.py

```diff
@@ -0,0 +1,123 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Lint as: python3
+"""TTC4900: A Benchmark Data for Turkish Text Categorization"""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import logging
+import os
+
+import datasets
+
+
+_DESCRIPTION = """\
+The data set is taken from kemik group
+http://www.kemik.yildiz.edu.tr/
+The data are pre-processed for the text categorization, collocations are found, character set is corrected, and so forth.
+We named TTC4900 by mimicking the name convention of TTC 3600 dataset shared by the study http://journals.sagepub.com/doi/abs/10.1177/0165551515620551
+"""
+
+_CITATION = ""
```

Comment:
is there a citation we can add here ?

Reply:
I will contact the original owner of the dataset and update it later.